cluster/aws/util.sh

```diff
@@ -352,6 +352,34 @@ function wait-for-instance-running {
   done
 }
 
+function allocate-elastic-ip {
+  $AWS_CMD allocate-address --domain vpc --output text | cut -f3
```

Comment:
Please quote your variables (through the change) as "${AWS_CMD}" with both quotation marks and curly braces.